angularjs-portal-home/src/main/webapp/my-app/marketplace/partials/marketplace.html

```diff
@@ -32,5 +32,34 @@
             ng-show="(portlets | filter:searchTermFilter | showApplicable:showAll | showCategory:categoryToShow).length > searchResultLimit"
            >Load More</button>
   </div>
+  
+  <div 
+    ng-show="(portlets | filter:searchTermFilter | showApplicable:showAll | showCategory:categoryToShow).length == 0" 
+    class="no-results">
+    <p>No matching MyUW content.</p>
+    <p>Maybe try:</p>
+    <ul>
+      <li ng-show="(portlets | filter:searchTermFilter | showApplicable:showAll).length > 0">
+        <a href="#" ng-click="selectFilter('az','')">Show matching content beyond the &quot;{{categoryToShow}}&quot; category</a>.</li>
+      <li 
+        ng-show="(portlets | filter:searchTermFilter | showCategory:categoryToShow).length > 0">
+         <a href="#" ng-click="toggleShowAll()">Show matching MyUW content even though it is not launchable</a>.</li>
+      <li>
+        <a ng-href="http://www.wisc.edu/directories/?q={{searchText}}"
+           target="_blank">Search the directory</a> (of people and offices) instead.</li>
+      <li>
+        <a ng-href="http://www.wisc.edu/search/?q={{searchText}}"
+           target="_blank">Search the wisc.edu domain on the Web</a> instead.</li>
+      <li>
+        <a ng-href="https://kb.wisc.edu/search.php?q={{searchText}}"
```

Comment:
It would be good to have these URLs in MISC_URLs, or maybe a new contant group. Thinking of master implementer..